lib/modules/manager/sbt/extract.ts

```diff
@@ -285,8 +291,31 @@ const query = q.tree<Ctx>({
 
 export function extractPackageFile(
   content: string,
-  _packageFile: string
+  packageFile: string
 ): PackageFileContent | null {
+  if (packageFile === 'project/build.properties') {
```

Comment:
```suggestion
  if (packageFile === 'project/build.properties' || packageFile.endsWith('/project/build.properties')) {
```
if it's inside a subfolder of the repo